index.html

```diff
@@ -1189,22 +1189,22 @@ <h2>Core Vocabulary Definitions</h2>
                 those defined in <code>securityDefinitions</code>.
                 These must all be satisfied for access to
                 resources.</td><td>mandatory</td><td><a href="http://www.w3.org/TR/2012/REC-xmlschema11-2-20120405/#string"><code>string</code></a> or <a>Array</a> of <a href="http://www.w3.org/TR/2012/REC-xmlschema11-2-20120405/#string"><code>string</code></a></td></tr>
-<tr class="rfc2119-table-assertion" id="td-vocab---Thing"><td><code></code></td><td>Set of named security configurations
+<tr class="rfc2119-table-assertion" id="td-vocab-securityDefinitions--Thing"><td><code>securityDefinitions</code></td><td>Set of named security configurations
                 (definitions only). Not actually applied unless
                 names are used in a <code>security</code>
-                name-value pair.</td><td>mandatory</td><td><a href="#securityscheme"><code>SecurityScheme</code></a> or <a>Array</a> of <a href="#securityscheme"><code>SecurityScheme</code></a></td></tr>
+                name-value pair.</td><td>mandatory</td><td><a>Map</a> of <a href="#securityscheme"><code>SecurityScheme</code></a></td></tr>
```

Comment:
That's a significant change for implementations.
How does this change correspond to the expected response and the issue that is addressed here?
It looks like an unrelated bug fix.

Reply:
How is backwards compatibility with TD1.0 guaranteed?

Reply:
This is a bug fix which was already fixed in a previous PR. I dont' know why this shows-up again but this change is correct and fulfill TD 1.0:

![image](https://user-images.githubusercontent.com/13832739/149123512-12cade74-d8cf-40c7-b6af-521e7516c591.png)